src/node_api_types.h

```diff
@@ -67,6 +67,7 @@ typedef enum {
   napi_generic_failure,
   napi_pending_exception,
   napi_cancelled,
+  napi_escape_called_twice,
```

Comment:
@jasongin can you think of any alternative to changing the enums to #defines ?  We definitely need to be able to add new errors to the list as we expand or update the api.